.travis.yml

```
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+- '8'
+- '10'
+cache:
+directories:
```

I think the tabulation is wrong. (te be verified)
```suggestion
     directories:
          - node_modules
```

---

done